backend/sheets/sheets.py

```diff
@@ -42,12 +42,19 @@ async def channel_description(self):
             backend.config.inktober_submit_channel
         )
 
-        topics = []
-        if now_day in backend.day_themes.day_themes.keys() and now_date.month == self.ink_month:
-            topics.append(f"{now_day}: {backend.day_themes.day_themes[now_day]}")
-
-        topic_str = f"Currently accepting - " + ", ".join(
-            topics) if topics else "Not accepting any submissions at this time"
+        topic = ""
+        triday = backend.triday_utils.get_triday(now_day)
+        triday_days = backend.triday_utils.get_days_from_triday(triday)
+        if triday == 11 and now_date.month == self.ink_month:
```

Comment:
Would it be an idea to remove all of the `and now_date.month == self.ink_month` and have it in a catch all above it for the inverse and have it return?